scripts/datasets/pretrain_corpus/prepare_bookcorpus.py

```diff
@@ -4,11 +4,18 @@
 import time
 import tarfile
 import argparse
-from gluonnlp.utils.misc import download
+from gluonnlp.utils.misc import download, load_checksum_stats
 
 _DOWNLOAD_URL\
     = "https://the-eye.eu/public/AI/pile_preliminary_components/books1.tar.gz"
 
+#_CURR_DIR = os.path.realpath(os.path.dirname(os.path.realpath(__file__)))
+#_URL_FILE_STATS_PATH = os.path.join(_CURR_DIR, '..', 'url_checksums', 'bookcorpus.txt')
+#_URL_FILE_STATS = load_checksum_stats(_URL_FILE_STATS_PATH)
+#print(_URL_FILE_STATS)
+
+#exit()
+

```

Comment:
Please remove the commented out code which is not needed in the public open source package